### PR TITLE
New Thunderdome 2

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Literally https://github.com/tgstation/TerraGov-Marine-Corps/pull/3304 but with mapmerge

## Why It's Good For The Game

It's a cool new Thunderdome

## Changelog
:cl: SunoPerson
add: New and revamped Thunderdome!
/:cl:

### CODE REVIEW

**Comment by spookydonut** [comment]
that commit history tho

**Comment by Rohesie** [comment]
dreamseeker_l1htQTmpal
Buttons are floating in the air and hard to access. Direction issue.

dreamseeker_Z0BJu8E8G4
Shutters start open.
Are these lights floating in the air intended?

File: _maps/map_files/generic/Admin_Level.dmm [hunk 1]
```diff
@@ -482,84 +290,38 @@
...
-"bD" = (
-/obj/machinery/computer/pod{
-	id = "thunderdomeaxe";
-	name = "Thunderdome Axe Supply"
+/obj/machinery/button/door/indestructible{
+	bound_y = 16;
+	id = "thunder1";
+	name = "Section 1 Podlocks";
+	pixel_x = 0;
+	pixel_y = -24;
```
Comment (Rohesie) [inline _maps/map_files/generic/Admin_Level.dmm]: Don't pixel edit it, please. Just set the correct direction.

File: _maps/map_files/generic/Admin_Level.dmm [hunk 2]
```diff
@@ -3,214 +3,73 @@
...
 "aB" = (
-/obj/structure/window/reinforced{
-	dir = 5
-	},
-/obj/effect/forcefield{
-	desc = "You can't get in. Heh.";
-	name = "Blocker"
+/obj/machinery/door/airlock/command{
+	name = "Thunderdome Administration";
+	req_access = null;
```
Comment (Rohesie) [inline _maps/map_files/generic/Admin_Level.dmm]: Can you fix this var edit?

File: _maps/map_files/generic/Admin_Level.dmm [hunk 3]
```diff
@@ -3198,12 +3151,34 @@
...
+	},
+/turf/open/floor/mainship,
+/area/tdome)
+"Ag" = (
+/turf/open/floor/plating/plating_catwalk,
+/area/tdome/tdomeadmin)
+"Aj" = (
+/obj/machinery/light,
+/obj/effect/forcefield{
+	desc = "You can't get in. Heh.";
```
Comment (Rohesie) [inline _maps/map_files/generic/Admin_Level.dmm]: Remove this var-edit, please.

**Comment by Rohesie** [comment]
Do test this locally, please. It seems to have broken movement.

**Comment by TiviPlus** [comment]
Closing this cus I'm having trouble unfucking my branch, might open a new PR later

**Pull request closed without merging**